app/routes/user.js

```diff
@@ -138,4 +147,24 @@ router.patch(
   userController.deactivateAccount
 )
 
+router.get(
+  '/auth/google',
+  isUnderMaintenance,
+  passport.authenticate('google', { scope: ['profile','email'], session: false })
+)
+
+router.get(
+  '/auth/google/callback',
+  isUnderMaintenance,
+  (req, res, next) => {
```

Comment:
please simplify the code and remove it from router

Reply:
Where should I put this? and what do I need to simplify?
Please let me know I didn't get this.

Reply:
hey, you are using this as a middleware so make it a middleware and place this in middleware folder

Reply:
Fixed this!